src/main/java/org/aksw/simba/squirrel/postprocessing/PostProcessor.java

```diff
@@ -0,0 +1,18 @@
+package org.aksw.simba.squirrel.postprocessing;
+
+/**
+ * A component to compute arbitrary things in a post processing style that should run concurrently.
+ * Logic can be implemented within the method {@link #postprocess()}.
+ */
+public interface PostProcessor extends Runnable {
```

Comment:
I do not really understand why this interface is necessary :thinking:

Reply:
@MichaelRoeder Last time we met, you said that post processing is a concept that could be needed for things other than deduplication and we should therefore generalize it by inventing an interface, for example :)

Reply:
Okay... so my old me had some idea with that. Thanks for reminding :grin: